p2p/peer.py

```diff
@@ -665,15 +665,12 @@ def add_peer(self, peer):
             # TODO: This is kept separate from the `expected_exceptions` to be
             # sure that we aren't silencing an error in our authentication
             # code.
-            self.logger.info('Got bad Ack during peer connection')
-            # We intentionally log this twice, once in INFO to be sure we don't
-            # forget about this and once in DEBUG which includes the stack
-            # trace.
-            self.logger.debug('Got bad Ack during peer connection', exc_info=True)
+            self.logger.info('Got bad auth ack from %s', repr(remote))
         except expected_exceptions as e:
-            self.logger.debug("Could not complete handshake with %s: %s", remote, repr(e))
+            self.logger.debug("Could not complete handshake with %s: %s", repr(remote), repr(e))
         except Exception:
-            self.logger.exception("Unexpected error during auth/p2p handshake with %s", remote)
+            self.logger.exception(
+                "Unexpected error during auth/p2p handshake with %s", repr(remote))
```

Comment:
Can also use `%r` instead of `%s` in all the changes above, instead of wrapping the arguments in `repr()`.